packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -203,8 +203,10 @@ export default class VscodeHatRenderer {
     }
 
     const svg = originalSvg
-      .replace(/fill="[^"]+"/, `fill="${color}"`)
-      .replace(/fill:[^;]+;/, `fill:${color};`);
+      .replace(/fill="none"/g, "")
```

Comment:
Does this not cause fill to be black? 

> If the fill attribute property is not specified, the default is black.

From [here](https://docs.aspose.com/svg/net/drawing-basics/fills-and-strokes/#:~:text=if%20the%20fill%20attribute%20property%20is%20not%20specified%2C%20the%20default%20is%20black.)

Reply:
It looks good in vscode, but I updated the regex to make it more secure